cli/src/main.rs

```diff
@@ -306,6 +306,11 @@ fn run() -> Result<(), CliError> {
                     .possible_values(&["json", "string"])
                     .requires("metadata")
                     .help("Set the encoding for the application metadata. Default value: string"),
+            )
+            .arg(
+                Arg::with_name("dry_run")
+                    .long("dry-run")
```

Comment:
Is there a good short version of this sort of thing?

Reply:
I did a quick search online and it seems `--dry-run` or `--dryrun` are often used.

Reply:
Added `-n` as short for `--dry-run`